.github/workflows/build-complete-samples.yml

```diff
@@ -416,6 +416,10 @@ jobs:
               name: 'bot-adaptive-card-actions'
               version: '6.0.x'
 
+            - project_path: 'samples/bot-all-cards/csharp/BotAllCards/BotAllCards.csproj'
```

Comment:
WHy BotAllCards.csproj is inside a folder?

Reply:
The majority of the samples were created entirely within the folder.
Reference Sample  - 
bot-adaptive-card-actions
bot-conversation-sso-quickstart
graph-proactive-installation ...etc